source/includes/fact-minimum-versions.rst

```diff
@@ -2,6 +2,13 @@ Starting in version 1.2, ``mongosync`` can sync between clusters running
 some older (pre-6.0) releases of MongoDB Server. This feature requires
 additional preparation and configuration in pre-6.0 releases.
 
-If you would like to migrate pre-6.0 versions of MongoDB Server
-using ``mongosync``, please `contact <https://mongodb.com/contact>`__
-your account team to inquire about Professional Services.
+Also, ``mongosync`` can perform a cross version migration
+from a lower major version source cluster to a higher major version
+destination cluster, up to two major versions ahead. For example,
+migrate from 6.0 to 7.0. This also requires additional
+preparation.
+
+If you would like to migrate pre-6.0 versions of MongoDB Server using
+``mongosync``, or perform a cross version migration, please `contact
+<https://mongodb.com/contact>`__ your account team to inquire about
+Professional Services.
```

Comment:
I think we can keep the original paragraph, since `If you would like to migrate pre-6.0 versions of MongoDB Server` also includes cross version migrate a pre-6.0 version.